ddtrace/utils/formats.py

```diff
@@ -65,3 +65,14 @@ def asbool(value):
         return value
 
     return value.lower() in ("true", "1")
+
+
+def flatten_dict(d, sep='.', prefix=''):
```

Comment:
TODO: Add a comment saying where this code came from, and then add notes for any edits you have made (if any).